config/goGenerate.coffee

```diff
@@ -0,0 +1,18 @@
+module.exports = (options, credentials) ->
```

Comment:
these variables are not only used in Klient so `generateKlientConfig` may be misleading.